### ISSUE TITLE
Add ability to query atlases with coordinate

### ISSUE THREAD

**Issue by rmarkello** [miykael/atlasreader#37]
It'd be nice to have some basic CLI or even just easily-exposed python functionality to query the available atlases for a given coordinate. Right now users are required to provide a statistical map, but it might be nice just to say "Hey, I wonder what anatomical region coordinate [x, y, z] is in the Harvard Oxford atlas" without opening up e.g., FSL or AFNI or whatnot.

This shouldn't take too much work, since `read_atlas_cluster()` basically already does this!

**Comment by rmarkello** [comment]
I can try and tackle this after #23 !

**Comment by rmarkello** [comment]
But any interested parties are encouraged to take this on, too! :+1:
